content/posts/2024-07-ruby-gsub-hashes/index.md

```diff
@@ -0,0 +1,38 @@
+---
+author: "George Bell"
+title: "TiL: Doing multiple replacements with gsub"
```

Comment:
Can you change this "TIL: Doing multiple replacements with gsub in Ruby"?

Reply:
Good shout, updated